draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1924,19 +1915,21 @@ CachedInformationType Values" registry that was defined in [RFC7924].
 
 ## Hash Algorithms    {#hash_algorithms}
 
-IANA is asked to establish a registry of hash algorithm values, initially
+IANA is asked to establish a registry of hash algorithm values, named
+"Hash algorithm numbers for Certificate Transparency logs" initially
 consisting of:
 
 |-------+---------------------|
 | Index | Hash                |
 |-------+---------------------|
-| 0     | SHA-256 [FIPS180-4] |
+| 0     | SHA-256 [RFC4634]   |
```

Comment:
4634 is Informational, but you've added it as a normative reference.  AIUI, we can only do informative references for Informational RFCs.